### DISCUSSION TITLE
Task 02 - Assignment 01

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#6] [Week 1 - Questions] [answered]
Currently, on 05/09/23, Week 1's Tasks 1-5 are available and visible in our class-repo. Next week, will the Tasks file contain only Week 2's Tasks making Week 1's Tasks no longer visible, or will all tasks over all weeks amalgamate in the Tasks folder?

**Comment by metasean** [comment, marked as answer]
Only Week 1 has it's own Task files.  Trying to do it all in one page was significantly more confusing, so I broke it up.  The rest of the weeks the instructions are much simpler (but to be clear, that does not mean they're easier.)
